### PR TITLE
feat: continue App component

### PR DESCRIPTION
**Pull request by matthew-chirgwin**
## Status
**READY**

### Commit Message Title
* When opening this PR, the raiser should set the title of this PR to the first line of their desired commit message, e.g:
```
feat: continue App component
```
* The reviewer should ensure that the first commit message field is of this form when performing the `squash and merge` from this page.

### Commit Message Description
```
- integrates consumer into App
- adds/updates existing tests to cover
- adds correct copy
- adds correct image and favicon
- fixes backend public file serving issue
- adds context for selected message, and integrates it into consumer

Contributes to: #60

Signed-off-by: Matthew Chirgwin <chirmatt@uk.ibm.com>
```
* The reviewer should copy the above text into the extended description field when performing the `squash and merge` from this page.

## Checklist
- [X] Automated tests exist
- [X] Documentation exists [link]()
- [X] Local unit tests performed
- [X] Sufficient logging/trace
- [X] Desired commit message set as PR title and commit description set above

### CODE REVIEW

File: ui/src/Bootstrap/App/App.scss [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+    }
+
+    &_heading {
+      @include carbon--type-style("productive-heading-04");
+      margin-bottom: $spacing-05;
+    }
+
+    &_body {
+
+    }
```
Comment (ajborley) [inline ui/src/Bootstrap/App/App.scss:23]: This empty selector can be removed?

File: ui/src/Panels/Consumer/Consumer.view.js [hunk 2]
```diff
@@ -66,19 +67,29 @@ const Consumer = (props) => {
...
-            message={
-              msg.status !== CONSTANTS.VERTX_ERROR_STATUS ? msg : undefined
-            }
-          />
+          <SelectedMessageConsumer key={`smc-${index}:${msg.index}`}>
+            {({ updateSelectedMessage, isSameAsSelected }) => {
+              return (
+                <ConsumerMessage
+                  {...idAttributeGenerator('consumer_consumed_message')}
+                  key={`consumed-message-${index}:${msg.index}`}
```
Comment (ajborley) [inline ui/src/Panels/Consumer/Consumer.view.js:75]: Does the `ConsumerMessage` still need a key prop if it's no longer a list item?

File: ui/src/Bootstrap/App/App.scss (hunk 1 above)
Comment (matthew-chirgwin) [reply ui/src/Bootstrap/App/App.scss:23]: I left it ready for styling :)

File: ui/src/Panels/Consumer/Consumer.view.js (hunk 2 above)
Comment (matthew-chirgwin) [reply ui/src/Panels/Consumer/Consumer.view.js:75]: You are right, it probably doesn't at this point - will look into it!

**Comment by ajborley** [comment]
Have been trying out the app against a real Kafka and can see messages getting happily consumed :)
Unfortunately the ordering of the consumed messages is a little odd:
image

**Comment by matthew-chirgwin** [comment]
> Have been trying out the app against a real Kafka and can see messages getting happily consumed :)
> Unfortunately the ordering of the consumed messages is a little odd:
> image

Good spot! How odd - 1st is correct, and then the rest are reversed. Will look into it.

**Pull request merged**